docs/EN_US/HPCCSystemAdmin/SA-Mods/DAFtslave.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE section PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+"http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
+<sect2 id="SA_SupportUtilities">
+  <title>Support Utilities</title>
+
+  <para>There are a few additional components which are neither system servers
+  or client interfaces but nonetheless important in supporting HPCC
+  tasks.</para>
+
+  <sect3 id="DAFilesrv">
+    <title>dafilesrv</title>
+
+    <para>The dafilesrv is a daemon that runs on system servers or physical
+    locations on the HPCC system. This daemon enables the HPCC components to
+    have file access to physical file locations. File access could be
+    spraying, despraying, copying, etc. It is also used to enable file
+    security.</para>
+
+    <para>The dafilesrv daemon runs as a process on every Thor or Roxie node.
```

Comment:
Not just Thor and Roxie nodes, all hpcc nodes may require it at some stage. Anything that requires a file from another node will go via dafilesrv - so e.g. hthor creates files access by others will require a dafilesrv on the eclagent nodes, esp accessing logs from any remote component will require a dafilesrv to be running on the node that/those components are running on.